app/Livewire/Project/Shared/EnvironmentVariable/Add.php

```
@@ -122,6 +123,66 @@ public function availableSharedVariables(): array
+                        try {
+                            $this->authorize('view', $service->server);
+                            $result['server'] = $service->server->environment_variables()
+                                ->pluck('key')
+                                ->toArray();
+                        } catch (\Illuminate\Auth\Access\AuthorizationException $e) {
+                            // User not authorized to view server variables
+                        }
+                    }
+                }
+            }
+        }
```

PHPMD (2.15.0)

The method availableSharedVariables uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)